ReactWindows/ReactNative.Shared/Modules/I18N/I18NUtil.cs

```diff
@@ -32,11 +39,19 @@ public static bool IsRightToLeftAllowed
         {
             get
             {
+#if WINDOWS_UWP
                 return (bool?)ApplicationData.Current.LocalSettings.Values[AllowRTL] ?? false;
+#else
+                return Boolean.Parse(ConfigurationManager.AppSettings[AllowRTL]);
+#endif
```

Comment:
Can you apply the bool.TryParse pattern that I changed in the DevInternalSettings.cs file?  I'd rather have a default to false for bad config data that a runtime exception.